.trunk/.gitignore

```diff
@@ -0,0 +1,9 @@
+*out
+*logs
+*actions
+*notifications
+*tools
+plugins
+user_trunk.yaml
+user.yaml
+tmp
```

Comment:
Do we need any of them currently? I'd only add things in case we really need it to keep it small and tidy.

Reply:
You're absolutely right. We don't need that. Thank you.